git_machete/github.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
```

Comment:
Hmmm actually, do we even need shebangs on any Python file, other than `git_machete/cmd.py` and the `git-machete` launcher script and in top-level folder?